.eslintrc.json

```diff
@@ -28,12 +30,34 @@
     "ecmaVersion": 2019,
     "sourceType": "module"
   },
+  "settings": {
+    "import/resolver": {
+      "typescript": {}
+    }
+  },
   "rules": {
     "no-console": "warn",
     "no-debugger": "error",
-    "import/order": "warn",
+    "sort-imports": "off",
+    "import/order": "off",
     "import/export": "off",
+    "import/first": "error",
+    "import/newline-after-import": "error",
     "import/no-extraneous-dependencies": "error",
+    "import/no-duplicates": "error",
+    "simple-import-sort/imports": [
+      "error",
+      {
+        "groups": [
+          ["^\\u0000", "/(fixtures|mocks|patches)$"],
+          ["^@?\\w"],
+          ["^raiden-\\w+"],
```

Comment:
Basically a nice idea to have it as separate group. But in fact it will always be a single import statement, right?

Reply:
Yes, usually, but we may change that in the future. I just didn't want it to go with `^` e.g. `@/` packages because then they get sorted later, and it gets ugly then.